process/block/displayBlock.go

```diff
@@ -50,7 +52,14 @@ func NewTransactionCounter(args ArgsTransactionCounter) (*transactionCounter, er
 		return nil, process.ErrNilMarshalizer
 	}
 
+	var log logger.Logger
+	log = logger.GetOrCreate("process/block")
+	if args.Logger != nil {
```

Comment:
is there any chance this would not come as expected here?

Reply:
no, old code, forgot to remove